CONTRIBUTING.md

```diff
@@ -31,7 +31,7 @@ The `develop` branch is the development branch which means it contains the next
 1. Create a PR against the `main` branch.
 1. After all tests pass and you have received approval from a CODEOWNER (including resolving any merge conflicts), merge the PR into `main`.
 1. Pull `main` locally, create a new tag, and push up.
-1. Confirm that the necessary assets are present in the newly created tag, and test on a WP install if desired.
+1. 1. (Optional) Download the zipped asset from the release and unzip locally, confirming that the expected files are present (and unexpected files, e.g. `composer.json`, Markdown files, etc. are omitted).
```

Comment:
```suggestion
1. 1. (Optional) Download the zipped asset from the release and unzip locally, confirming that the expected files are present (and unexpected files, e.g. files outlined in [.gitattributes](https://github.com/pantheon-systems/wp-native-php-sessions/blob/main/.gitattributes) are omitted).
```

Maybe we call out .gitattributes as a point of reference in case in the future we change what is shipped with the plugin (e.g. if/when we put these on packagist composer.json would be expected).